src/sentry/static/sentry/app/views/performance/landing/utils.tsx

```diff
@@ -139,27 +139,29 @@ export function getDefaultDisplayFieldForPlatform(
   return LandingDisplayField.ALL;
 }
 
-export const backendCardDetails = {
-  p75: {
-    title: 'Duration (p75)',
-    tooltip: 'Duration (p75)',
-    formatter: value => getDuration(value / 1000, value >= 1000 ? 3 : 0, true),
-  },
-  tpm: {
-    title: 'Throughput',
-    tooltip: 'Throughput',
-    formatter: formatAbbreviatedNumber,
-  },
-  failure_rate: {
-    title: 'Failure Rate',
-    tooltip: 'Failure Rate',
-    formatter: value => formatPercentage(value, 2),
-  },
-  apdex: {
-    title: 'Apdex',
-    tooltip: 'Apdex',
-    formatter: value => formatFloat(value, 4),
-  },
+export const backendCardDetails = (organization: LightWeightOrganization) => {
+  return {
+    p75: {
+      title: 'Duration (p75)',
+      tooltip: getTermHelp(organization, PERFORMANCE_TERM.P75),
+      formatter: value => getDuration(value / 1000, value >= 1000 ? 3 : 0, true),
+    },
+    tpm: {
+      title: 'Throughput',
```

Comment:
Should these be translated?

Reply:
Yes, that's a good call. I'm surprised we weren't already.